### ISSUE TITLE
[Rendering] Input.Time value is not received in the bot

### ISSUE THREAD

**Issue by bogdanbujdea** [MicrosoftDocs/msteams-docs#4512]
In some occasions the time selected inside an Input.Time control is not sent back to the bot. This happens only on the Teams desktop app and I noticed this behavior when the column for seconds was introduced. In the Teams web app this column does not exist(yet) and the Input.Time works as it should.

Steps to reproduce:

1. Send the adaptive card with the json below
2. On the Teams Desktop app click on "Ok"
3. The payload receive will look like this `{
  "startTime": "15:30"
}`
4. Change the date in the adaptive card
5. The payload will look like this `{}` 

JSON:
```
{
    "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
    "type": "AdaptiveCard",
    "version": "1.2",
    "body": [
        {
            "type": "TextBlock",
            "text": "What time do you want to meet?"
        },
        {
            "type": "Input.Time",
            "id": "startTime",
            "min": "09:00",
            "max": "17:00",
            "value": "15:30"
        }
    ],
    "actions": [
        {
            "type": "Action.Submit",
            "title": "OK"
        }
    ]
}
```

Also, is there a way to hide the seconds?
![image](https://user-images.githubusercontent.com/793611/140053289-fe0f4ab2-69d2-44ca-86bd-2777190f6d4c.png)

**Comment by Nivedipa-MSFT** [comment]
@bogdanbujdea - We are looking into this issue.We will update you.

**Comment by Nivedipa-MSFT** [comment]
@bogdanbujdea - I have tried the adaptive card on  Microsoft Teams Version 1.4.00.30066 on desktop.

![image](https://user-images.githubusercontent.com/92150070/140483008-e128a70e-530e-4b5e-9381-64de33611857.png)

Seconds are not showing.

**Comment by bogdanbujdea** [comment]
It seems this is an issue only on my machine because I changed the short time format to be "hh:mm:ss" instead of the default "hh:mm tt". The desktop app picks this app and shows the seconds in the input time, while also having issues sending the time back to the card on certain occasions. Once I changed back to the default time format and restarted Teams, the Input.Time format got back to normal and now it works every time.
I should also point out that I changed my time format because on Windows 11 the clock doesn't have the seconds anymore so there may be other Windows 11 users trying this and then having issues with the Input.Time control.
Sorry for the false alarm, I think we can close this.

![image](https://user-images.githubusercontent.com/793611/140485573-70710156-2793-4661-bc5c-9e240a8c200b.png)
